apps/portal/css/common.css

```diff
@@ -25,7 +25,15 @@
     text-align: right;
 }
 
-.ues-fullview-visible{
+.ues-fullview-visible { /* for the designer only */
+    position: absolute !important;
```

Comment:
Cannot remove the !important tag as this class needs to override some of the styles added by 3rd party libraries.